--- Pygments-2.2.0/pygments/lexers/lisp.py.orig	2017-01-22 22:01:32.000000000 +0100
+++ Pygments-2.2.0/pygments/lexers/lisp.py	2018-07-18 21:48:02.153318157 +0200
@@ -2327,13 +2327,13 @@
             token = Name.Function if token == Literal else token
             yield index, token, value
 
-        raise StopIteration
+        return
 
     def _process_signature(self, tokens):
         for index, token, value in tokens:
             if token == Literal and value == '}':
                 yield index, Punctuation, value
-                raise StopIteration
+                return
             elif token in (Literal, Name.Function):
                 token = Name.Variable if value.istitle() else Keyword.Type
             yield index, token, value
--- Pygments-2.2.0/pygments/lexers/sql.py.orig	2017-01-22 22:01:32.000000000 +0100
+++ Pygments-2.2.0/pygments/lexers/sql.py	2018-07-18 21:49:02.033317475 +0200
@@ -347,7 +347,10 @@
             # Emit the output lines
             out_token = Generic.Output
             while 1:
-                line = next(lines)
+                try:
+                    line = next(lines)
+                except StopIteration:
+                    return
                 mprompt = re_prompt.match(line)
                 if mprompt is not None:
                     # push the line back to have it processed by the prompt
